From 0bc91159fc37db617a59e60a2be61539eed7c83c Mon Sep 17 00:00:00 2001
From: Ken Wahid <kenx.wahid@intel.com>
Date: Thu, 23 Apr 2015 15:42:19 +0200
Subject: [PATCH] fdp_nfc: Fix coding error in resume/suspend functions

Fixed a small coding error that caused the fdp resume- and
suspend-to-RAM functions to return random values in some
circumstances.

Issue: IMINAN-33346

Change-Id: If1fa932a7f88139ada5c780391f92a7cd8163837
Tracked-On: https://jira01.devtools.intel.com/browse/AKP-1468
Signed-off-by: Ken Wahid <kenx.wahid@intel.com>
---
 drivers/staging/fdp/fdp_custom.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/staging/fdp/fdp_custom.c b/drivers/staging/fdp/fdp_custom.c
index 497d2ab02f87..6b8614b36115 100644
--- a/drivers/staging/fdp/fdp_custom.c
+++ b/drivers/staging/fdp/fdp_custom.c
@@ -828,8 +828,8 @@ int fdp_suspend(struct i2c_client *client, pm_message_t mesg)
 		return 0;
 	}
 
-	if (p_device->state == CUSTOM_OPENED)
-		/* disable_irq(p_device->irqout); */
+	/* if (p_device->state == CUSTOM_OPENED)
+		disable_irq(p_device->irqout); */
 
 	return 0;
 }
@@ -845,8 +845,8 @@ int fdp_resume(struct i2c_client *client)
 		return 0;
 	}
 
-	if (p_device->state == CUSTOM_OPENED)
-		/* enable_irq(p_device->irqout); */
+	/* if (p_device->state == CUSTOM_OPENED)
+		enable_irq(p_device->irqout); */
 
 	return 0;
 }
-- 
1.9.1

